### PR TITLE
Additional changes to use commands instead of actuator_pprz

### PR DESCRIPTION
**Pull request by tmldeponti**
Some additional changes to the merged PR Use Commands instead of actuator_pprz (https://github.com/paparazzi/paparazzi/pull/3265)

### CODE REVIEW

File: sw/airborne/firmwares/rotorcraft/autopilot_firmware.c [hunk 1]
```diff
@@ -92,7 +93,13 @@ bool WEAK autopilot_ground_detection(void) {
 bool WEAK autopilot_in_flight_end_detection(bool motors_on UNUSED) {
   if (autopilot_in_flight_counter > 0) {
     /* probably in_flight if thrust, speed and accel above IN_FLIGHT_MIN thresholds */
-    if ((stabilization.cmd[COMMAND_THRUST] <= AUTOPILOT_IN_FLIGHT_MIN_THRUST) &&
+    /** Select the correct available Thrust Setting*/
+#ifdef COMMAND_THRUST
+    float thrust_level = commands[COMMAND_THRUST];
+#else
+    float thrust_level = stabilization.cmd[COMMAND_THRUST];
```
Comment (gautierhattenberger) [inline sw/airborne/firmwares/rotorcraft/autopilot_firmware.c:100]: I don't get this. If `COMMAND_THRUST` is not defined, the `else` case will fail.

File: sw/airborne/firmwares/rotorcraft/autopilot_firmware.c [hunk 2]
```diff
@@ -305,7 +312,13 @@ void autopilot_check_in_flight(bool motors_on)
       /* if thrust above min threshold, assume in_flight.
        * Don't check for velocity and acceleration above threshold here...
        */
-      if (stabilization.cmd[COMMAND_THRUST] > AUTOPILOT_IN_FLIGHT_MIN_THRUST) {
+/** Select the correct available Thrust Setting*/
+#ifdef COMMAND_THRUST
+float thrust_level = commands[COMMAND_THRUST];
+#else
+float thrust_level = stabilization.cmd[COMMAND_THRUST];
+#endif
```
Comment (gautierhattenberger) [inline sw/airborne/firmwares/rotorcraft/autopilot_firmware.c:320]: same here

**Review by gautierhattenberger — CHANGES_REQUESTED** [review]

File: sw/airborne/firmwares/rotorcraft/autopilot_firmware.c (hunk 1 above)
Comment (tmldeponti) [reply sw/airborne/firmwares/rotorcraft/autopilot_firmware.c:100]: indeed this is wrong, i corrected it

File: sw/airborne/firmwares/rotorcraft/autopilot_firmware.c (hunk 2 above)
Comment (tmldeponti) [reply sw/airborne/firmwares/rotorcraft/autopilot_firmware.c:320]: yes true!

**Review by gautierhattenberger — APPROVED** [review]

**Pull request merged**